content/docs/install/plugins.md

```diff
@@ -16,15 +16,3 @@ autocmd! BufNewFile,BufRead Dvcfile,*.dvc setfiletype yaml
 ```
 
 to your `~/.vimrc`(to be created if it doesn't exist).
-
-## IntelliJ IDEs
-
-A community member, [@prihoda](https://github.com/prihoda), maintains a plugin
-for IntelliJ IDEs, it offers a more robust integration than just syntax
-highlighting.
-
-You can download the plugin from
-[JetBrains Plugins repository](https://plugins.jetbrains.com/plugin/11368-dvc-support-poc)
-
-For more information, visit the plugin's repository:
-[iterative/intellij-dvc/](https://github.com/iterative/intellij-dvc/)
```

Comment:
Removed since it's outdated but opened iterative/intellij-dvc/issues/7 about it.